extensions/aris/src/bigDataCluster/tree/addControllerTreeNode.ts

```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import { TreeItem, TreeItemCollapsibleState } from 'vscode';
+import { NodeInfo } from 'azdata';
```

Comment:
By convention we tend to import * as azdata and use azdata.NodeInfo. Same for vscode.

Reply:
fixed.